plugins/quickbooks-online/src/src/Facade/QuickBooksFacade.php

```diff
@@ -371,123 +370,121 @@ function (array $a, array $b) {
             $this->logger->notice(sprintf('Invoice credit amount is : %s', $ucrmPayment['creditAmount']));
             if ($ucrmPayment['creditAmount'] > 0) {
-		try {
-			$theResourceObj = Payment::create(
-			[
-			'CustomerRef' => [
-				'value' => $qbClient->Id,
-				'name' => $qbClient->DisplayName,
-				],
-			'TotalAmt' => $ucrmPayment['creditAmount'],
-			'TxnDate' => substr($ucrmPayment['createdDate'],0,10),
-			]
-			);
-		$response = $dataService->Add($theResourceObj);
-		if ($response instanceof IPPIntuitEntity) {
-			$this->logger->info(
-			sprintf('Payment ID: %s credit exported successfully.', $ucrmPayment['id'])
-			);
-		}
-		if (! $response) {
-			$this->logger->info(
-			sprintf('Payment ID: %s credit export failed.', $ucrmPayment['id'])
-		);
-		}
-		if ($response instanceof \Exception) {
-			throw $response;
-		}
-		$this->handleErrorResponse($dataService);
+                try {
+                    $theResourceObj = Payment::create(
+                        [
+                            'CustomerRef' => [
+                                'value' => $qbClient->Id,
+                                'name' => $qbClient->DisplayName,
+                            ],
+                            'TotalAmt' => $ucrmPayment['creditAmount'],
+                            'TxnDate' => substr($ucrmPayment['createdDate'], 0, 10),
+                        ]
+                    );
+                    $response = $dataService->Add($theResourceObj);
+                    if ($response instanceof IPPIntuitEntity) {
+                        $this->logger->info(
+                            sprintf('Payment ID: %s credit exported successfully.', $ucrmPayment['id'])
+                        );
+                    }
+                    if (! $response) {
+                        $this->logger->info(
+                            sprintf('Payment ID: %s credit export failed.', $ucrmPayment['id'])
+                        );
+                    }
+                    if ($response instanceof \Exception) {
+                        throw $response;
+                    }
+                    $this->handleErrorResponse($dataService);
 
-            } catch (\Exception $exception) {
-                $this->logger->error(
-                    sprintf(
-                        'Payment ID: %s export failed with error %s.',
-                        $ucrmPayment['id'],
-                        $exception->getMessage()
-                    )
-                );
+                } catch (\Exception $exception) {
+                    $this->logger->error(
+                        sprintf(
+                            'Payment ID: %s export failed with error %s.',
+                            $ucrmPayment['id'],
+                            $exception->getMessage()
+                        )
+                    );
+                }
             }
-	    }
-            $pluginData->lastExportedPaymentID = $ucrmPayment['id'];
```

Comment:
Hey @raymondjclark, I did some cleanup of this class and the change you did first was actually almost correct, it just had to be moved a single level up - not with whitespace but under the `}`, so I changed it back and applied the fix.